source/ingress.go

```diff
@@ -152,7 +160,11 @@ func endpointsFromIngress(ing *v1beta1.Ingress) []*endpoint.Endpoint {
 			continue
 		}
 
-		endpoints = addEndpointsFromTargetAnnotation(ing, rule.Host, endpoints)
+		endpoints = getEndpointsFromTargetAnnotation(ing, rule.Host)
```

Comment:
@claytononeill it is okay, it was just a suggestion on to improve overall code structure - which can be done in subsequent PR. Otherwise this pull request looks okay to me. If no objections I will have this merged /cc @linki